lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -4916,25 +4879,25 @@ class ConvertTritonGPUToLLVM
 void ConvertTritonGPUToLLVM::initSharedMemory(
     size_t size, TritonGPUToLLVMTypeConverter &typeConverter) {
   ModuleOp mod = getOperation();
-  OpBuilder b(mod.getBodyRegion());
+  OpBuilder rewriter(mod.getBodyRegion());
```

Comment:
We shouldn't rename this into rewriter.
ConversionPatternRewriter is inherited from OpBuilder and different in concept. And here it's definitely a OpBuilder not a Rewriter.

I know the change is for utilizing the macros, though. I tend just to keep this func as it was to avoid misunderstanding.